src/MCPClient/debian/copyright

```diff
@@ -1,6 +1,6 @@
 This work was packaged for Ubuntu by:
 
-    Justin Simpson <jsimpson@artefactual.com>
+    Artefactual Systems Inc. <sysadmin@artefactual.com>
```

Comment:
It should be sysadmin@ in all places, if there are individual emails left or ops@ anywhere, it is because I missed it, I will update the PR.